### DISCUSSION TITLE
How to build the ORT-Genai from source?

### DISCUSSION THREAD

**Discussion by prashantaithal** [microsoft/onnxruntime-genai#760] [Q&A]
How to build the C++ APIs ORT-Genai  library from source and install them?
[Documentation](https://onnxruntime.ai/docs/genai/howto/build-from-source.html#install-cc-header-file-and-library)  only mentions how to build onnxruntime but not onnxruntime-genai from source.

**Comment by kunal-vaishnavi** [comment]
The documentation has now been updated there.
